sklearn/calibration.py

```diff
@@ -515,7 +515,8 @@ def predict(self, T):
 
 
 def calibration_curve(y_true, y_prob, normalize=False, n_bins=5):
-    """Compute true and predicted probabilities for a calibration curve.
+    """Compute true and predicted probabilities for a calibration curve,
+     for binary classifications.
```

Comment:
We usually have a short one-line summary at the beginning of a docstring. Could you add your helpful change after a blank line, please?